src/styles/main.scss

```diff
@@ -1,7 +1,22 @@
 @import "utils";
 @import "fonts";
 @import "typography";
+@import "blocks/header";
+@import "blocks/phone";
+@import "blocks/menu";
+@import "blocks/recommended";
+@import "blocks/categories";
+@import "blocks/buy-reasons";
+@import "blocks/contact";
+@import "blocks/footer";
+@import "blocks/button-wrapper";
+
+html {
+  scroll-behavior: smooth;
+}
 
 body {
-  background: $c-gray;
+  font-weight: 700;
+
+  @include set-font-Inter;
```

Comment:
It's better to use a more descriptive mixin name like 'set-Inter-font' instead of 'set-font-Inter'.

Reply:
I changed name to `set-font-inter`.